source/includes/markdown/_14-publish-your-app.html.md

```diff
@@ -68,28 +68,49 @@ The explorer is not meant to be an exhaustive tool with every endpoint in the AP
+<img src="../images/submit-my-test-app.png" />
 
-* Be aware of the [OWASP Top 10 Web App Security Risks](https://owasp.org/www-project-top-ten/) when developing your app. 
+From there, you'll review an interactive "pre-flight" checklist of app features before submitting an app. This includes:
 
-* If you are new to app security, we recommend reading [The Web Application Hacker's Handbook](https://www.amazon.com/The-Web-Application-Hackers-Handbook/dp/1118026470).
+- Adding an app icon that uniquely represents your app
+- Setting up [OAuth](/docs/oauth) by adding a redirect URL
+- Adding app images that show what your app can do
+- Adding a short description to summarize what your app can do at a glance
+- Adding a longer description (at least 250 characters) to tell users about the features of your app
+- Adding information about your company to build trust with users
+- Adding a support page URL so that users can seek help
+- Adding your app’s privacy policy URL
 
-* Use end-to-end encryption. Use a trusted site to test your TLS security (such as: https://www.ssllabs.com/ssltest/).
+Along with building or providing the above features, you'll be able to submit additional app details through the submit form,
+such as setup instructions, a screencast video link, and other listing details.
 
-* Join the [Asana developer community](https://forum.asana.com/c/developersAPI). This is the best resource to get technical questions answered as well as get notified about new API features, deprecations, and other breaking changes. 
+### App review process
 
-* Once your app is listed, you can answer Asana user questions in the [integrations section of the Asana community](https://forum.asana.com/c/integrations).
+Once your app has been submitted through the [developer console](https://app.asana.com/0/my-apps), the review process will
+begin. At this point, the the app will no longer be open for further edits. Note that if your app uses
+[App Components](/docs/overview-of-app-components), your submission will go through a security review and QA process
+before it’s listed in the in-product [app gallery](/docs/installing-an-app).
 
-### Updating your existing app listing
-If you wish to update your existing listing, send an email to api-support at asana.com with the specific changes. 
+You can track the status of your app submission on the same **Submit for review** page. If your app submission passes requirements,
+you'll receive an email notification that your app is ready to publish. At this point, you can select the **Publish app** button in 
+the developer console to share your app:
 
-We expect existing apps to maintain the same level of user experience that they had when accepted to the apps directory.  
+<img src="../images/approved-app.png" />
+
+If your app submission doesn't meet requirements, you will receive communications regarding the next steps (i.e., any changes required)
```

Comment:
Do we want to be more specific about the communication method? Like via email?

Reply:
Sounds good to me